syft/cataloging/license.go

```diff
@@ -1,15 +1,46 @@
 package cataloging
 
-import "github.com/anchore/syft/internal/licenses"
+import (
+	"github.com/anchore/syft/internal/licenses"
+)
+
+// IncludeLicenseContent controls whether license content should be included.
+type IncludeLicenseContent string
+
+const (
+	IncludeLicenseContentAll     IncludeLicenseContent = "all"
```

Comment:
3088 is really about trying to add license content for cases where it was found in the value and moved to the content field and using the scanner to fill in `value` with an ID where possible. I haven't looked over the full PR yet, but do we want to introduce `all` now in this PR?

Reply:
I can remove this - It was part of my notes from the design session. Since it was the default case post package cataloging tasks(not modifying the contents) it felt easy to add, but I can remove it to reduce scope.

Reply:
Sync review we decided this was a free addition to we're keeping it in this PR